mithril-aggregator/src/services/certifier.rs

```diff
@@ -50,6 +50,11 @@ pub enum CertifierServiceError {
     #[error("Open message for beacon {0:?} already certified.")]
     AlreadyCertified(SignedEntityType),
 
+    /// The open message is expired, no more single signatures may be
+    /// attached to it nor be certified again.
+    #[error("Open message for beacon {0:?} is expired.")]
```

Comment:
I'm not sure about the use of "beacon" here, as there is only a "beacon" in Snapshots.
Can we add the notion of "signed_entity_type"?